lib/components/ColorCard/ColorCard.tsx

```diff
@@ -0,0 +1,51 @@
+export interface ColorCardProps {
+  /** Title text shown on the card */
+  title: string;
+  /** Content text shown on the card */
+  content?: string;
+  /** Action content shown on the card */
+  actionContent?: string;
+  /** Background color of the card */
+  backgroundColor?: string;
+}
+
+export const ColorCard = ({ title, content, actionContent, backgroundColor = '#444BACF2' }: ColorCardProps) => (
+  <>
+    {actionContent ? (
+      <BaseCard title={title} content={content} actionContent={actionContent} backgroundColor={backgroundColor} />
+    ) : (
+      <a href="/" className="flex rounded-[28px] outline-none transition-transform hover:scale-105 focus:scale-105">
+        <BaseCard title={title} content={content} actionContent={actionContent} backgroundColor={backgroundColor} />
+      </a>
+    )}
+  </>
+);
+
+const BaseCard = ({ title, content, actionContent, backgroundColor }: ColorCardProps) => {
+  const Heading = actionContent ? 'h1' : 'h2';
+  return (
+    <div
+      className={`relative flex flex-col gap-[24px] rounded-[28px] px-[32px] pb-[64px] pt-[32px] text-white`}
+      style={{ backgroundColor }}
+    >
+      <Heading className="overflow hyphens-auto text-pretty text-[28px] font-bold leading-[34px]">{title}</Heading>
+      {content && <p>{content}</p>}
+      {actionContent ? (
+        <a
+          href="/"
+          className="absolute bottom-0 right-[32px] translate-y-2/4 rounded-[40px] outline-none transition-transform hover:scale-105 focus:scale-105"
+        >
+          <div
+            className="flex select-none gap-4 rounded-[40px] bg-white px-[40px] py-[20px] text-[28px] font-bold leading-[38px]"
+            style={{ color: backgroundColor }}
+          >
+            {actionContent}
+            <i>→</i>
+          </div>
+        </a>
+      ) : (
+        <i className="absolute bottom-[14px] right-[32px] select-none text-[28px] font-bold leading-[32px]">→</i>
```

Comment:
Why using `<i>` for the icon?

On a sidenote VoiceOver seems funny and reads "arrow" for this, but did not stop it even with these:
```
aria-label=""
aria-hidden="true"
```

Reply:
We will use SVG when it becomes available. Bootstrap and Font Awesome use `<i />` as an alternative. Semantically, `<i />` should be ok to use with icons. I add ARIA hidden.